### ISSUE TITLE
💡 You can easily convert docs from adoc to html to pdf

### ISSUE THREAD

**Issue by damithc** [nus-cs2103-AY1718S1/forum#197]
As you know CS2103/T (and CS2101) **final submissions require pdf files** while most of your current docs are in `.adoc` format.

💡 Instructions on how to automatically convert from `adoc` to `html`, and from `html` to `pdf`, are given in the [developer guide](https://nus-cs2103-ay1718s1.github.io/addressbook-level4/DeveloperGuide.html#documentation).

💡 If you don't want to set up auto-publishing, you can convert from `adoc` to `html` manually using gradle, as explained [here](https://nus-cs2103-ay1718s1.github.io/addressbook-level4/UsingGradle.html#rendering-asciidoc-files)

❗️ **Try to preserve hyperlinks in your pdf files** if you use some other tool/method to generate pdf files. You are going to be using those pdf files later in the week 13 testathon and it will be easier if the links in the pdf files are clickable.

**Comment by yunpengn** [comment]
Suggest using this ruby gem [asciidoctor-pdf](https://github.com/asciidoctor/asciidoctor-pdf) to directly convert `adoc` to `pdf`

**Comment by damithc** [comment]
> Suggest using this ruby gem asciidoctor-pdf to directly convert adoc to pdf

Yup, that's a better way, if you already have Ruby on your computer and familiar with the usage. Otherwise a bit excessive to install a new language just for this :-p
